db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2018_06_04_211453) do
+ActiveRecord::Schema.define(version: 2018_06_15_155436) do
```

Comment:
Did you run another migration after the one included in this PR? The version timestamp will normally match the filename of the latest migration that's been run on the DB, so locally I get `2018_06_13_173211` after running the migration.